packages/next/client/future/image.tsx

```diff
@@ -380,34 +412,71 @@ export default function Image({
       // instead console.error(ref) during mount.
       unoptimized = true
     } else {
-      if (typeof widthInt === 'undefined') {
-        throw new Error(
-          `Image with src "${src}" is missing required "width" property.`
-        )
-      } else if (isNaN(widthInt)) {
-        throw new Error(
-          `Image with src "${src}" has invalid "width" property. Expected a numeric value in pixels but received "${width}".`
-        )
-      }
-      if (typeof heightInt === 'undefined') {
-        throw new Error(
-          `Image with src "${src}" is missing required "height" property.`
-        )
-      } else if (isNaN(heightInt)) {
-        throw new Error(
-          `Image with src "${src}" has invalid "height" property. Expected a numeric value in pixels but received "${height}".`
-        )
-      }
-      if (!VALID_LOADING_VALUES.includes(loading)) {
-        throw new Error(
-          `Image with src "${src}" has invalid "loading" property. Provided "${loading}" should be one of ${VALID_LOADING_VALUES.map(
-            String
-          ).join(',')}.`
-        )
+      if (fill) {
+        if (width) {
+          throw new Error(
```

Comment:
My take is that if someone is adding `height` and `width` to an image in `fill` mode, 90%+ of the time it's going to be because they are making an error--they think that it will do something, and they're going to be confused about why it isn't having any effect. 

I'd think the case of having your own wrapper automatically providing height and width probably wouldn't need `fill` mode to begin with, or if it did the abstraction could be modified to provide `height/width` or `fill` but not both.